.eslintrc

```diff
@@ -187,7 +187,7 @@
     "space-return-throw-case": [2],
     "space-unary-ops": [2, {"words": true, "nonwords": false}],
     "spaced-line-comment": [0, "always"],
-    "strict": [2, "never"],
+    "strict": 0,
```

Comment:
That's true. I usually do my best to avoid inline annotations. One of the
reasons - changing linter requires removing them all over the code.

I reckon it should be ok in this case. Only two files with awkwardness